script/release/Dockerfile

```diff
@@ -1,24 +1,15 @@
 FROM golang:1.18.7 AS build
 
-ARG CONTAINERD_VERSION=1.5.7
-ARG NYDUS_VERSION=v2.1.5
+ARG NYDUS_VERSION=v2.1.6
```

Comment:
Maybe also update the version for all test actions.

Reply:
This is the last un-updated ```v2.1.5```. Others is ```v2.1.6```.